packages/hash/frontend/src/pages/[account-slug]/types/entity-type/edit-bar.tsx

```diff
@@ -96,32 +97,34 @@ const EditBarContents = ({
 const useFreezeScrollWhileTransitioning = () => {
   const observerRef = useRef<ResizeObserver | null>(null);
   const ref = useRef<HTMLDivElement>(null);
+  const editBarContext = useContext(EditBarContext);
```

Comment:
`useContext(FooBarContext)` is a bit too raw. I prefer seeing `useFooBar` as it makes code more readable. We can add checks for `undefined` into this custom hook and throw if the hook is called outside the context provider. Alternatively, we can return some default context too. If we do this, hook's return signature becomes simpler and the component ends up easier to read.

This is probably too minor for blocking this PR, but I'm happy to re-review it if you want to make changes.

Reply:
Completely agree - this happened as the context had been in this file, and then I refactored. Will fix.